geode-core/src/main/java/org/apache/geode/cache/asyncqueue/internal/AsyncEventQueueFactoryImpl.java

```diff
@@ -281,6 +281,12 @@ public AsyncEventQueueFactory setForwardExpirationDestroy(boolean forward) {
     return this;
   }
 
+  // keep this method internal
+  public AsyncEventQueueFactory setPartitionedRegionClearUnsupported(boolean supported) {
+    gatewaySenderAttributes.partitionedRegionClearUnsupported = supported;
```

Comment:
The method argument and field names here are confusing, as the field name is tracking whether PR clear *is not* supported, but the method argument name implies that it is setting whether PR clear *is* supported.

Reply:
fixed